src/Articles.jsx

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ArticleList } from "./ArticleList";
+import Lottie from "lottie-react";
+import loading from "./assets/loading.json";
+import { PageSelection } from "./PageSelection";
+import { Filters } from "./Filters";
+
+export const Articles = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [page, setPage] = useState(1);
+  const [totalItems, setTotalItems] = useState(0);
+  const [filters, setFilters] = useState({
+    order: "desc",
+    sortBy: "created_at",
+    topic: "",
+  });
+  const limit = 4;
+
+  useEffect(() => {
+    setIsLoading(true);
+    axios
+      .get(
+        `https://dhiran-news.onrender.com/api/articles?topic=${filters.topic}&sort_by=${filters.sortBy}&order=${filters.order}&limit=${limit}&p=${page}`
+      )
+      .then(({ data }) => {
+        setArticles(data.articles);
+        setTotalItems(data.total_count);
+        setIsLoading(false);
+      });
+  }, [page, filters]);
```

Comment:
This looks good, but I would recommend extracting this out and using Axios's instance as we will be making a lot of requests to this api. This is also not how Axios handles queries either so do have a read of the notes/docs for this.

Notes for this:
l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance